packages/core/data/oot/world_mq/deku_tree_mq.yml

```
@@ -18,27 +18,67 @@
     "MQ Deku Tree Slingshot Room Far Chest": "can_use_sticks || has_fire"
     "MQ Deku Tree Basement Chest": "can_use_sticks || has_fire"
     "MQ Deku Tree GS Lobby Crate": "can_damage_skull"
+    "MQ Deku Tree Grass Entrance Lower 1": "can_cut_grass"
+    "MQ Deku Tree Grass Entrance Lower 2": "can_cut_grass"
+    "MQ Deku Tree Grass Entrance Lower 3": "can_cut_grass"
+    "MQ Deku Tree Grass Entrance Lower 4": "can_cut_grass"
+    "MQ Deku Tree Grass Entrance Lower 5": "can_cut_grass"
+    "MQ Deku Tree Grass Entrance Upper 1": "can_cut_grass"
+    "MQ Deku Tree Grass Entrance Upper 2": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Front 1": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Front 2": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Back 1": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Back 2": "can_cut_grass"
+    "MQ Deku Tree Grass Room Before Compass 1": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Room Before Compass 2": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Room Before Compass 3": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Room Before Compass 4": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Room Before Compass 5": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Room Before Compass 6": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Room Before Compass 7": "can_cut_grass && (can_use_bow || can_use_sticks || can_use_din)"
+    "MQ Deku Tree Grass Basement Lower 1": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Lower 2": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Lower 3": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Lower 4": "can_cut_grass"
+    "MQ Deku Tree Grass Room Before Spike 1": "can_cut_grass && (can_use_slingshot || can_use_bow) && (can_use_sticks || has_fire)"
+    "MQ Deku Tree Grass Room Before Spike 2": "can_cut_grass && (can_use_slingshot || can_use_bow) && (can_use_sticks || has_fire)"
+    "MQ Deku Tree Grass Room Before Spike 3": "can_cut_grass && (can_use_slingshot || can_use_bow) && (can_use_sticks || has_fire)"
+    "MQ Deku Tree Grass Room Before Spike 4": "can_cut_grass && (can_use_slingshot || can_use_bow) && (can_use_sticks || has_fire)"
```

iirc, this is the first room in deku basement. if you can use bow, you dont need sticks or fire cause of the torch. i imagine this was copy/pasted from water room access which would then need sticks or fire for the torches within.
```suggestion
    "MQ Deku Tree Grass Room Before Spike 4": "can_cut_grass && (can_use_bow || (can_use_slingshot && (can_use_sticks || has_fire))"
```